kitsune/users/models.py

```diff
@@ -71,8 +72,8 @@ class Profile(ModelBase, SearchMixin):
                            verbose_name=_lazy(u'Biography'))
     website = models.URLField(max_length=255, null=True, blank=True,
                               verbose_name=_lazy(u'Website'))
-    twitter = models.URLField(max_length=255, null=True, blank=True,
-                              verbose_name=_lazy(u'Twitter URL'))
+    twitter = models.CharField(max_length=15, null=True, blank=True, validators=[TwitterValidator],
+                               verbose_name=_lazy(u'Twitter Handle'))
```

Comment:
This should probably say "Twitter Username", since username is a more common term.